sgt-back-end/index.js

```diff
@@ -0,0 +1,168 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+  from "grades";
+  `;
+  db
+    .query(sql)
+    .then(result => {
+      res.status(200).json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({ error: 'An unexpected error occurred' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  if (!(('course' in req.body) && ('name' in req.body) && ('score' in req.body))) {
+    res.status(400).json({ error: 'Please make sure to include a course, name, and score' });
+  } else {
+    if (!!req.body.course && !!req.body.name) {
+      if (req.body.score >= 0 && req.body.score <= 100) {
```

Comment:
Please verify that `score` is an integer as well. Fractional `score`s should fail with a `400 Bad Request`:

---

![image](https://user-images.githubusercontent.com/16143771/119598206-61981400-bd97-11eb-8484-99a3c01d6a76.png)